proxy/grpc/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
```

Comment:
1. It would have been easier to review this if the factoring out of the generated gRPC bindings was done in a separate PR.

2. I think `api` or `proxy-api` is a better name for this than `conduit-grpc`.